package/boot/uboot-envtools/files/ath79

```diff
@@ -52,7 +52,8 @@ glinet,gl-ar150)
 	ubootenv_add_uci_config "/dev/mtd1" "0x0" "0x8000" "0x10000"
 	;;
 netgear,wndr3700|\
-netgear,wndr3700-v2)
+netgear,wndr3700-v2|\
+netgear,wndrmac-v2)
```

Comment:
I have a wndr3800 running ath79-generic (OpenWrt master) which is not listed here. So either this entry is not needed or the wndr3800 is missing.

Reply:
I'm compiling a new payload with this line (and the corresponding v1, for the new commits) removed. I'll be uploading the new commits as soon as I validate this.

Reply:
The wndr3800 is missing. However, that should be fixed separately if desired is not relevant here.

Reply:
adding entries to uboot-envtools in the past was only accepted if the device actually needed to access the uboot envs at all. (like to reset a boot counter for failed boots in dual-partition devices or to read MAC address from uboot envs)

Reply:
All right then, I'll leave this in. Consider this PR as ready from my side.